ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/CustomerCenterBottomSheet.kt

```diff
@@ -0,0 +1,62 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.material3.SheetState
+import androidx.compose.material3.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.revenuecat.purchases.ui.revenuecatui.ExperimentalPreviewRevenueCatUIPurchasesAPI
+
+/**
+ * Composable offering a bottom sheet Customer Center UI configured from the RevenueCat dashboard.
+ */
+@ExperimentalPreviewRevenueCatUIPurchasesAPI
+@ExperimentalMaterial3Api
```

Comment:
I'm propagating the required annotations to the developer, since we're using google's experimental APIs (even though it's the recommended API 😅)

Reply:
Hmm, this means we're risking binary incompatibility if the Material3 version we expect is different from the one we get at runtime. We basically need to always update to the latest Material3 version (until this is stable), and thereby force our consumers to do the same. 

Do we need the bottomsheet composable? It is fairly trivial for people to wrap the `CustomerCenterView` in a `ModalBottomSheet`. We could even provide sample code in our docs.

Reply:
Well we don't *need* it... So it would be:
```
if (isCustomerCenterBottomSheetVisible) {
    val customerCenterSheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)

    CustomerCenterBottomSheet(
        onDismissRequest = { isCustomerCenterBottomSheetVisible = false },
        sheetState = customerCenterSheetState,
    )
}
```
vs 
```
if (isCustomerCenterBottomSheetVisible) {
    val customerCenterSheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true)

    ModalBottomSheet(
        onDismissRequest = { isCustomerCenterBottomSheetVisible = false } ,
        modifier = Modifier.fillMaxHeight(...), // Optional, can customize how it looks
        sheetState = sheetState,
    ) {
        CustomerCenter(modifier = Modifier.fillMaxSize()) // Or any other modifiers...
    }
}
```

So a custom composable would give us more control of the display and I think it's slightly nicer (I'm still a bit reticent to leave total control to the developer 😅) but I see what you mean, and I don't have a solution... so yeah I think we should hold on this and see what devs think. As you said, we can try to document this. 

cc @vegaro @aboedo in case you have other thoughts.

Reply:
Yea it would definitely be a nice addition! I'm just not sure if that weighs up against the binary incompatibility risk. We can also decide to add it as soon as the `ModalBottomSheet` API becomes stable?

Reply:
Yup that sounds like a good idea 👍